compiler/frontend/src/mir/body.rs

```diff
@@ -411,7 +411,53 @@ impl BodyBuilder {
             responsible,
         )
     }
+    pub fn push_if_not<E>(
+        &mut self,
+        hir_id: &hir::Id,
+        condition: Id,
+        else_builder: E,
+        responsible: Id,
+    ) -> Id
+    where
+        E: FnOnce(&mut Self),
+    {
+        self.push_if_else(
+            hir_id,
+            condition,
+            |body| {
+                body.push_nothing();
+            },
+            else_builder,
+            responsible,
+        )
+    }
+    pub fn push_is_bool(&mut self, hir_id: &hir::Id, value: Id, responsible: Id) -> Id {
+        let is_condition_true = self.push_equals_value(value, true, responsible);
```

Comment:
This should be called something like `is_value_true` to match the generalized HIR ID below